plugins-compat-tester/src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -197,4 +202,39 @@ public void testCustomWarPackagerVersions() {
 		assertTrue("No matches",m.matches());
 	}
 
+	@Test
+	public void testBom() throws IOException, PlexusContainerException, PomExecutionException, XmlPullParserException {
+		ImmutableList<String> includedPlugins = ImmutableList.of(
+			"workflow-api", // From BOM
+			"accurev"	// From Update Center
+		);
+
+		PluginCompatTesterConfig config = new PluginCompatTesterConfig(testFolder.getRoot(),
+				new File("../reports/PluginCompatReport.xml"), 
+				new ClassPathResource("m2-settings.xml").getFile());
+
+		File bomFile = new ClassPathResource("jenkins-bom.xml").getFile();
+		config.setBom(bomFile);
+		config.setIncludePlugins(includedPlugins);
```

Comment:
This test class needs `config.setExternalMaven(<whatever-location>)` i.e.: `config.setExternalMaven(new File("/usr/bin/mvn"));` but based on: https://github.com/jenkinsci/plugin-compat-tester/pull/194 I let this without that config update.